src/components/Button.tsx

```diff
@@ -0,0 +1,11 @@
+import Button from '@mui/material/Button';
+
+const BasicButtons = ({text}:any) => {
+  return (
+    <Button variant="contained">
+      {text}
+    </Button>
+  );
+}
+
+export default BasicButtons;
```

Comment:
export default cause tree shaking in React, try to avoid this if it is possible

Reply:
better is
export const BasicButtons =

Reply:
done